.github/ISSUE_TEMPLATE/installation_issue.md

```diff
@@ -0,0 +1,27 @@
+---
+
+name: Installation Issue 
+about: Ask about installing geopandas 
+title: ""
+labels: "installation"
```

Comment:
We don't currently have this label, but I can add this if we're explicitly supporting installation issue questions via the issue tracker.

Reply:
That may even be good to add to the template itself. `I have looked through [installation] issues in the repo.`.